stable/opsportal/templates/deployment-proxy.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
What about adding if logic around this and create a value to enable it; default the value to disabled and override that value in the Addon? That way when we have proper authorization we can stop using the bearerproxy.